lib/MergeStream.js

```diff
@@ -47,6 +47,10 @@ MergeStream.prototype.addConnectionsStream = function (connectionsStream) {
     }
   });
 
+  this._connectionsStreams[index].on('request', function() {
+    self.emit('request');
+  });
+
```

Comment:
I'm not going to merge this: csa.js should be oblivious about what comes in; Why do you need request to be emitted here? For all csa.js cares, there is just a stream that comes in.

If you need to count it in your demo, as I understand why you want to have an event emitted, just keep count it using the connection stream itself, not the result stream.